.travis.yml

```diff
@@ -31,6 +31,11 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - pip install python-ldap
```

Comment:
What about use `requirements.txt` file in root project path https://github.com/OCA/maintainer-quality-tools/pull/311?